lua/tardis/interiors/default.lua

```diff
@@ -71,7 +71,7 @@ T.Interior = {
 		default_screen             = {pos = Vector(42, 0.75, 27.1),                   ang = Angle(0, -5, 0),                  },
 		default_screenbutton       = {pos = Vector(44.5, 9.75, 38.2),                 ang = Angle(215, 85, 90),               },
 		default_wibblylever        = {pos = Vector(-48, 18, 5.4),                     ang = Angle(335, 347, 6),               },
-		default_enginerelease      = {pos = Vector(-61.229, 21.622, -4.514),          ang = Angle(90, 90.911, -90),           },
+		default_enginerelease      = {pos = Vector(-61.229, 21.622, -4.514),          ang = Angle(180, 90.911, -90),           },
```

Comment:
nitpick: new number pushes the alignment out of the closing bracket

Reply:
Fixed